api/snapshot.go

```diff
@@ -182,6 +182,8 @@ func (s *Server) SnapshotBackup(w http.ResponseWriter, req *http.Request) (err e
 		labels[types.KubernetesStatusLabel] = string(kubeStatus)
 	}
 
+	labels[types.LonghornVolumeAccessModeLabel] = string(vol.Spec.AccessMode)
```

Comment:
Suggest adding the label in the mutating webhook rather than in the API.
If user creates a backup by operating CR, then the lable won't be added in current implementation.

Reply:
Thanks, I will follow the suggestion to make the operations through mutating webhooks.